entrypoint.sh

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env bash
+mkdir -p /app/configs/
+cp -n /app/default-configs/* /app/configs/
+mkdir -p /app/vanity/
+cp -n /app/default-vanity/* /app/vanity/
+cd /app/ && ./MasterServer
```

Comment:
Should this file have `set -e` at the start?

Reply:
Potentially, won't hurt to add it